### PR TITLE
Add font and font size

### PR DESCRIPTION
**Pull request by cob16**

### CODE REVIEW

File: app/views/pdfs/show.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+body {
+  font: normal 17px Liberation Sans, sans-serif;
```
Comment (asmega) [inline app/views/pdfs/show.css]: not needed now also i lack the knowledge. may be worth checking what unit is best of pdf rendering ie is it pixels or other unit?

File: app/views/pdfs/show.html.erb [hunk 2]
```diff
@@ -1,22 +1,33 @@
-<h2><%= @sub_heading %></h2>
-<h1><%= @heading %></h1>
+<% content_for :head do %>
+  <style type="text/css">
+    <%= render template: "pdfs/show.css" %>
```
Comment (asmega) [inline app/views/pdfs/show.html.erb]: should this be a partial?

File: app/views/pdfs/show.html.erb [hunk 3]
```diff
@@ -1,22 +1,33 @@
...
 
-<% @sections.each do |section| %>
-  <% if section[:summary_heading] %>
-    <h2><%= section[:summary_heading] %><h2>
-    <hr>
-    <p><%= section[:heading] %></p>
-    <p></p>
+<% content_for :body do %>
+  <h2 class="subheading"><%= @sub_heading %></h2>
+  <h1><%= @heading %></h1>
```
Comment (asmega) [inline app/views/pdfs/show.html.erb]: how come the h1 comes second?

Comment (cob16) [reply app/views/pdfs/show.html.erb]: h2 is smaller, one of the strange things of this template is that the subheading is first

File: app/views/pdfs/show.css (hunk 1 above)
Comment (sequencemedialimited) [reply app/views/pdfs/show.css]: What are the standard units/sizes in GDS?

File: app/views/pdfs/show.html.erb [hunk 4]
```diff
@@ -1,22 +1,33 @@
...
-  <% else %>
-    <h2><%= section[:heading] %></h2>
-    <hr>
-    <p></p>
-  <% end %>
+  <% @sections.each do |section| %>
+    <% if section[:summary_heading] %>
+      <h2><%= section[:summary_heading] %>
+        <hr class="dark">
+        <h2><%= section[:heading] %></h2>
```
Comment (sequencemedialimited) [inline app/views/pdfs/show.html.erb]: Is that intentionally an `<h2/>` followed by another `<h2 />`?

File: app/views/pdfs/show.html.erb [hunk 5]
```diff
@@ -1,22 +1,33 @@
...
+      <h2><%= section[:heading] %></h2>
+      <hr class="dark">
+    <% end %>
 
-  <% section[:questions].each do |question| %>
-    <p><%= question[:label] %>: <%= question[:answer] %></p>
-    <hr>
-    <p></p>
+    <% section[:questions].each do |question| %>
+      <div>
```
Comment (sequencemedialimited) [inline app/views/pdfs/show.html.erb]: `<div />` or `<p />`?

**Pull request merged**